src/components/tooltip/Tooltip.tsx

```diff
@@ -5,6 +5,8 @@ import getProp from 'lodash/get';
 import TetherComponent from 'react-tether';
 
 import TetherPosition from '../../common/tether-positions';
+// @ts-ignore flow import
+import FocusTrap from '../focus-trap';
```

Comment:
should we limit this to GuideTooltip or could we see a use case for all tooltips?

are all GuideTooltips considered a dialog?

Reply:
I don't think it's limited to GuideTooltips, the tooltip that the issue arose on was not a Guide Tooltip. I don't know if all GuideTooltips are considered dialogs, but I feel like it makes sense to enable this FocusTrap on a case by case basis, rather than enable it for all. What do you think?

Reply:
oh you're right, i got my tooltips mixed up and thought all FTUX tooltips were GuideTooltips

Reply:
although I think it's a little weird that USM isn't using the GuideTooltip component

Reply:
I believe GuideTooltips are usually multi-step tooltips